skelebot/systems/generators/yaml.py

```diff
@@ -23,15 +35,13 @@ def loadConfig(env=None):
         config = Config()
     else:
         values = {}
+        values[VERSION_ATTRIBUTE] = loadVersion()
         for attr in yamlData.keys():
```

Comment:
You can also load through both keys and values with `.items()` instead of `.keys()`